public/js/main.js

```diff
@@ -13,19 +13,27 @@ var displayResults = function(data) {
         "</div></a></div>");
     });
   } else {
-    $("#analysis").html("No result found");
+    $("#results").append("<div class='notFound'>No result found</div>");
   }
 }
+var submitQuery = function() {
+  $("#analysis").empty();
```

Comment:
Analysis is not there anymore. Meanwhile, it seems that #result isn't cleaned every research (or I miss the call)